tests/acceptance/features/recursion_error.feature

```diff
@@ -0,0 +1,1090 @@
+Feature: Checking Django failure
+
+  Scenario Outline: A plain scenario
+    Given I run a step
```

Comment:
Can you use an outline title and/or a step name that explain the intention of the test? (e.g. "... will cause a recursion error when ...")